### DISCUSSION TITLE
2023-07-14

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#38] [daily_meeting]
### 오늘의 할 일
- 공통
    - ~~뽀모도로 템플릿 질문 확정(노아, 룩소)~~
    - ~~세부 정책 결정하기~~
    - ~~인스턴스 생성 및 cd 구축(엽토, 마코)~~
    - ~~API 다듬기~~
- 각자
    - ~~맡은 부분 이슈 발행~~
    - ~~개발 전 준비단계~~
   
- 기술블로그에 기획 변경된 이유 작성하는건 어떤지 논의

### 내일의 할 일 
- 공통
  - 왼손과의 커피챗(청소시간 이후) 
  - 각자 개발

### 언젠가 해야할 일
- 기획 변경된 이유 정리

**Comment by woosung1223** [comment]
### 체크인
9점 - 아침에 핫식스 원샷 때리고 와서 팔팔합니다.

**Comment by jaehee329** [comment]
### 체크인
7점 - 아침에 일어나기 쉽지 않았어요. 내일은 주말이니까 끝날 때는 오를 듯..?ㅎㅎㅎ
### 체크아웃
8점 - 주말 직전이라 +1 드립니다.

**Comment by nlom0218** [comment]
### 체크인

9점 - 벌써 금요일여서 -1 / 컨디션은 좋음!

### 체크아웃

10점 - 오랜만에 히이로랑 점심을 먹어서 좋았다.

**Comment by MoonJeWoong** [comment]
### 체크인

9점 : 예비군 끝나서 기분이 너무 좋습니다 ㅎㅎ... 이번 주 있었던 사항들 동기화가 되면 완벽한 10점이 될 수 있을 것 같아요~

### 체크아웃
10점 : 한 주간 진행되었던 일들에 대해서 많이 동기화할 수 있었던 하루였던 거 같습니다! 앞으로 공부해야할게 정말 많네요~

**Comment by woo-jk** [comment]
### 체크인
7점 - 날씨가 너무나 꿉꿉해요. 🌧️🌧️🌧️🌧️🌧️🌧️🌧️🌧️

### 체크아웃
10점 - 개발 들어가서 잼나다👍 내일 여행가서 기분 좋타

**Comment by aak2075** [comment]
### 체크인

8점 - ktx연착(-1) 지옥철 사람 많아서 다음꺼 타서(-1)

### 체크아웃

10점 - cicd 파이프라인 완성해서 좋아요

**Comment by yeopto** [comment]
### 체크인

8점 - 아커 집에서 쾌적한 숙면을 해서 너무 좋지만 습해서 -2

### 체크아웃

8점 - 엄청 정신이 없어서 기분이 같다고 느껴져요
